includes/definitions/alliedtelesis.yaml

```diff
@@ -1,10 +1,14 @@
-os: allied
-text: AlliedWare
+os: alliedtelesis
```

Comment:
is aw+ the os/firmware name? If so, the actual OS for us should be that - maybe `awplus`